### PR TITLE
goimports everything

### PR DESCRIPTION
**Pull request by silasdavis**

### CODE REVIEW

File: docs/generator.go [hunk 1]
```diff
@@ -7,13 +7,13 @@ import (
 	"strings"
 	"text/template"
 
-	docs "github.com/eris-ltd/eris-db/docs/generator"
 	commands "github.com/eris-ltd/eris-db/cmd"
+	docs "github.com/eris-ltd/eris-db/docs/generator"
```
Comment (zramsay) [inline docs/generator.go]: no space needed

File: docs/generator.go [hunk 2]
```diff
@@ -7,13 +7,13 @@ import (
...
 
 	clientCommands "github.com/eris-ltd/eris-db/client/cmd"
+	"github.com/eris-ltd/eris-db/definitions"
 	"github.com/eris-ltd/eris-db/version"
 	"github.com/spf13/cobra"
```
Comment (zramsay) [inline docs/generator.go]: space missing

File: logging/loggers/channel_logger_test.go [hunk 3]
```diff
@@ -3,8 +3,9 @@ package loggers
 import (
 	"testing"
```
Comment (zramsay) [inline logging/loggers/channel_logger_test.go]: rm empty line

File: manager/eris-mint/state/state.go [hunk 4]
```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/tendermint/go-wire"
 
 	core_types "github.com/eris-ltd/eris-db/core/types"
-	"github.com/tendermint/tendermint/types"
 	"github.com/eris-ltd/eris-db/util"
+	"github.com/tendermint/tendermint/types"
```
Comment (zramsay) [inline manager/eris-mint/state/state.go]:
```
import (
    "go"
    "deps"
    "are"
    "here"

    "repo"
    "deps"
    "are"
    "here"

    "glided"
    "deps"
    "are"
    "here"
)
````

**Comment by benjaminbollen** [comment]
@silasdavis can you also then gofmt everything; because I keep needing to not-commit gofmt changes since logging

File: util/hell/cmd/hell/main.go [hunk 5]
```diff
@@ -6,15 +6,15 @@ import (
...
 
+	"github.com/Masterminds/glide/action"
 	"github.com/Masterminds/glide/cache"
 	"github.com/Masterminds/glide/cfg"
 	"github.com/Masterminds/glide/msg"
 	"github.com/Masterminds/glide/path"
+	"github.com/Masterminds/glide/repo"
 	"github.com/Masterminds/glide/util"
 	"github.com/eris-ltd/eris-db/util/hell"
 	"github.com/spf13/cobra"
```
Comment (zramsay) [inline util/hell/cmd/hell/main.go]: split import paths per above template

**Comment by silasdavis** [comment]
@benjaminbollen goimports gofmts everything

**Comment by benjaminbollen** [comment]
LGTM

**Pull request merged**